product-tours/tour-sklearn/Tour-with-Sklearn.ipynb

```diff
@@ -0,0 +1,771 @@
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "neptune.stop()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "tests"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# check logs\n",
+    "correct_logs_set = {'charts_sklearn'}\n",
+    "for name in ['precision', 'recall', 'fbeta_score', 'support']:\n",
+    "    for i in range(10):\n",
+    "        correct_logs_set.add('{}_class_{}_test_sklearn'.format(name, i))\n",
+    "from_exp_logs = set(exp.get_logs().keys())\n",
+    "assert correct_logs_set == from_exp_logs, '{} - incorrect logs'.format(exp)\n",
+    "\n",
+    "# check sklearn parameters\n",
+    "assert set(exp.get_properties().keys()) == set(gbc.get_params().keys()), '{} parameters do not match'.format(exp)\n",
+    "\n",
+    "# check neptune parameters\n",
+    "assert set(exp.get_parameters().keys()) == set(parameters.keys()), '{} parameters do not match'.format(exp)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "ta6VcI8z6DvM",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Automatic logging to Neptune: summary"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "You just learned how to log scikit-learn classification summary to Neptune using single function.\n",
+    "\n",
+    "Click on the link that was outputted to the console or [go here](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-312/charts) to explore an experiment similar to yours. In particular check:\n",
+    "\n",
+    "* [logged classifier parameters](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-312/details) as properties,\n",
+    "* [logged pickled model](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-312/artifacts?path=model%2F&file=estimator.skl),\n",
+    "* [logged test predictions](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-312/artifacts?path=csv%2F&file=test_predictions.csv),\n",
+    "* [logged test predictions probabilities](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-312/artifacts?path=csv%2F&file=test_preds_proba.csv),\n",
+    "* [logged test scores](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-312/charts),\n",
+    "* [logged classifier visualizations](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-312/logs) - look for \"charts_sklearn\",\n",
+    "* [logged metadata](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-312/details) including git summary info."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## If you want to learn more, go to the [Neptune documentation](https://docs.neptune.ai/integrations/sklearn.html)."
```

Comment:
I don't thikn this link works as it is in the header

Reply:
Similarly to the pytorch tour, I checked this one and link works correctly.

I think we can keep it like this.